spec/system/flexible_page_spec.rb

```diff
@@ -0,0 +1,37 @@
+RSpec.describe "FlexiblePage" do
+  let(:base_path) { "/flexible-page" }
+
+  before do
+    ENV["ALLOW_LOCAL_CONTENT_ITEM_OVERRIDE"] = "true"
+    stub_const("ContentItemLoader::LOCAL_ITEMS_PATH", "spec/fixtures/local-content-items")
+  end
+
+  after do
+    ENV["ALLOW_LOCAL_CONTENT_ITEM_OVERRIDE"] = nil
+  end
+
+  describe "GET <flexible-page>" do
+    it "has a meta description tag" do
+      visit base_path
+
+      expect(page).to have_css('meta[name="description"][content="This is the history of 10 Downing Street"]', visible: :hidden)
+    end
+
+    it "renders a page_title flexible section" do
+      visit base_path
+
+      expect(page).to have_selector("h1", text: "10 Downing Street")
+      expect(page).to have_text("History")
```

Comment:
I think a test is needed for lead paragraph if it exists.

Reply:
Added